elastic-stack/README.md

```diff
@@ -6,23 +6,23 @@
 
 ### DESCRIPTION
 
-Create links from SignalFx charts and alerts to any URL-addressable destination such as Kibana.
+Create links from Splunk Infrastructure Monitoring charts and alerts to any URL-addressable destination such as Kibana.
 
 The links appear in the Action menus adjacent to metric property values displayed in dashboards and alerts.
 
 The metric property values are used as query parameter and/or path variable values of the URL link.
 
 ### INSTALLATION
 
-Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination. 
+Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination.
 
 Read about the possible configurations [here](https://docs.signalfx.com/en/latest/dashboards/dashboard-links.html) and priority linking [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-and-global-data-links).
```

Comment:
Unless this feature allows local data linking, or linking to a dashboard, Splunk, or Kibana, this sentence is unnecessary.